services/core-api/app/api/projects/project/project_util.py

```diff
@@ -6,8 +6,11 @@
 
 class ProjectUtil:
 
-    def notifiy_file_updates(project, mine):
-        if is_feature_enabled(Feature.MINE_APPLICATION_FILE_UDPATE_ALERTS):
+    @staticmethod
+    def notify_file_updates(project, mine, status_code = None):
+        status_codes = ["SUB", "ASG", "CHR"]
```

Comment:
the staticmethod here seems odd, staticmethod in python is a classmethod (instead of an instance method). 

the first parameter is usually the class itself and is provided automatically.

Reply:
Well, here's my reasoning, and feel free to let me know where I'm going wrong!

- Might not be obvious that this class is ProjectUtils and not Project
- before it was neither static nor class method, and it was complaining that the first parameter (project) should be renamed to self, which was a hint to me that something was _terribly wrong_.
- when choosing between class and static methods, my shorthand is "do I need to access the instance of the class (ie: the self parameter)" and yes = class and no = static method. This class has no attributes and is really just a (very small) collection of functions, so maybe there's a better way to set that up.